src/Test/Perf/util/runner_util.csx

```diff
@@ -38,3 +40,134 @@ async Task<ScriptState<object>> RunFile(string fileName)
     return await state.ContinueWithAsync<object>(text, scriptOptions);
 }
 
+/// Gets all csx file recursively in a given directory
+IEnumerable<string> GetAllCsxRecursive(string directoryName)
```

Comment:
At this point, we dont use `skip`. If we happen to use it later, we can introduce it with minimal code change.